specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/2017-11-03-preview/examples/HanaInstances_Get.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Over the holidays, we updated the structure of this repository. Moving the `2017-11-03-preview` folder into a folder called `preview` would fix things. e.g.

`Microsoft.HanaOnAzure/2017-11-03-preview` -> `Microsoft.HanaOnAzure/preview/2017-11-03-preview`
 
For more information on what we did over the holiday, you can take a look at the wiki: 
[azures-rest-api-specs/wiki/December-2017-Refactoring](https://github.com/Azure/azure-rest-api-specs/wiki/December-2017-Refactoring)

Reply:
Got it!